namespaces/live-1.cloud-platform.service.justice.gov.uk/offender-categorisation-dev/resources/messaging.tf

```diff
@@ -0,0 +1,69 @@
+module "risk_profiler_change" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-sqs?ref=3.2"
+
+  environment-name       = "${var.environment-name}"
+  team_name              = "${var.team_name}"
+  infrastructure-support = "${var.infrastructure-support}"
+  application            = "${var.application}"
+  sqs_name               = "risk_profiler_change"
+
+  redrive_policy = <<EOF
+  {
+    "deadLetterTargetArn": "${module.risk_profiler_change_dead_letter_queue.sqs_arn}","maxReceiveCount": 1
+  }
+  EOF
+
+  providers = {
+    aws = "aws.london"
+  }
+}
+
+resource "aws_sqs_queue_policy" "risk_profiler_change_policy" {
+  queue_url = "${module.risk_profiler_change.sqs_id}"
+
+  policy = <<EOF
+  {
+    "Version": "2012-10-17",
+    "Id": "${module.risk_profiler_change.sqs_arn}/SQSDefaultPolicy",
+    "Statement":
+      [
+        {
+          "Effect": "Allow",
+          "Principal": {"AWS": "*"},
+          "Resource": "${module.risk_profiler_change.sqs_arn}",
+          "Action": "SQS:SendMessage"
```

Comment:
I think you might need to add a condition here, as per this other SQS definition:
https://github.com/ministryofjustice/cloud-platform-environments/blob/master/namespaces/live-1.cloud-platform.service.justice.gov.uk/cccd-staging/resources/messaging.tf#L95

Reply:
Thanks @digitalronin. I did look at the other example, they are writing to the queue from a sns topic - which needed a bit more work. I don't think I need a condition here as I just need to let the service send to the queue.  Would it be possible to merge so I could test?  Probably need to iterate a few times

Reply:
I'll also contact the people you suggested - thanks